test/lib/build/TaskRunner.js

```diff
@@ -2,6 +2,7 @@ import test from "ava";
 import sinonGlobal from "sinon";
 import esmock from "esmock";
 import logger from "@ui5/logger";
+logger.setLevel("perf");
```

Comment:
why do we have to set the log level here?

Reply:
In function `_executeTask`, we have some code that is only executed when log-level "perf" is activated